### DISCUSSION TITLE
not working for mirrored installation

### DISCUSSION THREAD

**Discussion by steinpath** [Sithuk/ubuntu-server-zfsbootmenu#43] [General]
**sorry, when I run the script to make a mirrored installation, it stops (just a bit before the full execution) at this point:**

+ for file in /etc/logrotate.d/*
+ grep -Eq '(^|[^#y])compress' /etc/logrotate.d/unattended-upgrades
+ sed -i -r 's/(^|[^#y])(compress)/\1#\2/' /etc/logrotate.d/unattended-upgrades
+ for file in /etc/logrotate.d/*
+ grep -Eq '(^|[^#y])compress' /etc/logrotate.d/wtmp
+ reinstate_apt
+ '[' yes = yes ']'
+ cp /mnt/ub_server/etc/apt/sources.list /mnt/ub_server/tmp/
+ sed -i s,http://archive.ubuntu.com/ubuntu/,http://archive.ubuntu.com/ubuntu,g /mnt/ub_server/etc/apt/sources.list
+ mv /mnt/ub_server/etc/apt/sources.list.bak /mnt/ub_server/tmp/
+ '[' -f /mnt/ub_server/etc/apt/apt.conf.d/30apt_error_on_transient ']'
+ mv /mnt/ub_server/etc/apt/apt.conf.d/30apt_error_on_transient /mnt/ub_server/tmp/
+ keyboard_console
+ chroot /mnt/ub_server
root@ubuntu:/# 
root@ubuntu:/# #dpkg-reconfigure --priority=medium --frontend=dialog keyboard-configuration console-setup && setupcon #Priority needs to be set to medium or low to trigger console-setup dialog.
root@ubuntu:/# 
root@ubuntu:/# export DEBIAN_PRIORITY=high
root@ubuntu:/# export DEBIAN_FRONTEND=dialog
root@ubuntu:/# 
root@ubuntu:/# dpkg-reconfigure keyboard-configuration

**and the command line has this command(or part of a command), and it is blocked:**

ubuntu@ubuntu:~/ubuntu-server-zfsbootmenu$ 'r/log.var/log.'loglS-nficell-ffon.' onfolub_offol///rp

**no matter if I set encrypet/not encrypted: just the same exit.

in attachments: syslog and script with my variables**

[bin bash Script installs.txt](https://github.com/Sithuk/ubuntu-server-zfsbootmenu/files/15443478/bin.bash.Script.installs.txt)
[syslog.txt](https://github.com/Sithuk/ubuntu-server-zfsbootmenu/files/15443479/syslog.txt)

**Comment by Sithuk** [comment]
Are you sure the script didn't finish successfully? The command line typically looks like that when I test the script. When you restart the system what happens?

**Reply by steinpath** [reply]
well, when i see the disks in in gparted they look as a complete installation effectively happened. anyway they don't boot, I get the uefi panel. I tried setting different boot entries, with no result. may be It need some other uefi settings? 
thanks

**Reply by steinpath** [reply]
maybe there are some troubles with ReFind?

**Comment by Sithuk** [comment]
I've just tested a 2 disk mirror setup with 22.04 on both server and desktop. Both worked fine. Although I did get a package download error with one of the mirrors so I turned off use mirrors and it worked fine.

**Reply by steinpath** [reply]
Well, in fact the problem is with reFind: it doesn't start and uefi can't see it. as a matter of fact now even Fedora, that I had installed with reFind, can't boot.
any suggestions to reactivate reFind?

**Reply by Sithuk** [reply]
If you can get to an UEFI shell then try manually starting rEFInd.

**Reply by steinpath** [reply]
the point is just this: my uefi shell shows no rEFInd.

**Reply by steinpath** [reply]
ok: rEFInd reactivated: now the mirrored system works ✌️
it takes a lot of time to be loaded, but it works.

**Reply by Sithuk** [reply]
That's good news. Thank you for the feedback.
